package.json

```diff
@@ -3,13 +3,15 @@
   "version": "1.0.0",
   "description": "Shine blog framework",
   "scripts": {
+    "add-baseurl":"node hugo/lib/getUrl.js",
     "build": "npm run webpack-build && npm run hugo-build",
-    "build-for-staging": "npm run clear-public && npm run clear-content && npm run fetch-content -- --staging && npm run build",
+    "build-for-staging": "npm run clear-public && npm run clear-content && npm run add-baseurl && npm run fetch-content -- --staging && npm run build",
     "build-for-production": "npm run clear-public && npm run clear-content && npm run fetch-content -- --production && npm run build",
     "clear-content": "cd hugo/lib && ./clear-content.sh",
     "clear-public": "rm -rf ./public",
     "fetch-content": "node hugo/lib/getArticles.js",
-    "hugo-build": "hugo --config=config/hugo.config.toml --destination=public/",
+    "hugo-build": "npm run add-baseurl &&hugo --config=config/hugo.config.toml --destination=public/",
```

Comment:
👾  This is minor, but just for the sake of readability we can add a space between `&&` and `hugo`